depmapomics/loading.py

```diff
@@ -721,27 +721,28 @@ def load(samplesetname, workspaces,
   if len(noarxspan) > 0:
     print("we found "+str(len(noarxspan))+" samples without arxspan_ids!!")
     noarxspan = noarxspan.sort_values(by = 'stripped_cell_line_name')
-    dfToSheet(samples, "depmap ALL samples not found", creds)
+    # dfToSheet(samples, "depmap ALL samples not found", creds)
```

Comment:
no it is important to use these spreadsheets (all 3 of them)

Reply:
resolved in new PR